pkg/auth/auth.go

```diff
@@ -114,17 +115,17 @@ func (u *UserAuth) Validate() error {
 	return u.k8sAuth.Validate()
 }
 
-func (u *UserAuth) resolve(groupVersion, kind string) (string, error) {
+func (u *UserAuth) resolve(groupVersion, kind string) (string, bool, error) {
 	resourceList, err := u.k8sAuth.GetResourceList(groupVersion)
 	if err != nil {
-		return "", nil
+		return "", false, nil
```

Comment:
Why if the previous call is returning an error we are not bubbling it up it but instead we return error=nil?